packages/utils/src/lib/reports/__snapshots__/report-diff-improved.md

```diff
@@ -0,0 +1,48 @@
+# Code PushUp
+
+🥳 Code PushUp report has **improved** – compared target commit `0123456` with source commit `abcdef0`.
+
+## 🏷️ Categories
+
+|🏷️ Category|⭐ Current score|⭐ Previous score|🗠 Score change|
```

Comment:
The score change emoji is barely visible on my machine. 😢 
Could we replace it by something else, e.g. one of these? 🚀🔄🌱📈
![image](https://github.com/code-pushup/cli/assets/6306671/12f0312d-740f-41e9-a1dc-fc7284be6b46)

Reply:
Replaced with 🔄.